internal/store/model/receiver.go

```diff
@@ -20,6 +20,19 @@ type Receiver struct {
 	UpdatedAt      time.Time           `db:"updated_at"`
 }
 
+type ExpandedReceiver struct {
+	ID                   uint64              `db:"id"`
+	Name                 string              `db:"name"`
+	Type                 string              `db:"type"`
+	Labels               pgc.StringStringMap `db:"labels"`
+	Configurations       pgc.StringAnyMap    `db:"configurations"`
+	Data                 pgc.StringAnyMap    `db:"-"` //TODO do we need this?
+	ParentID             sql.NullInt64       `db:"parent_id"`
+	CreatedAt            time.Time           `db:"created_at"`
+	UpdatedAt            time.Time           `db:"updated_at"`
+	ParentConfigurations pgc.StringAnyMap    `db:"parent_configurations"`
+}
```

Comment:
Why do we need to create another model here?
We could just put the merged configurations in `configurations` right?